### ISSUE TITLE
Implement VisB SVG objects list

### ISSUE THREAD

**Issue by leuschel** [hhu-stups/prob-issues#111]
The VisB JSON file can now also contain a list of SVG objects that are dynamically added to the SVG.

For this we need to implement a ProB kernel command for this Prolog predicate:
 prob2_visb_svg_objects(List)

and then similar to prob_prolog add SVG objects via Javascript commands in the ProB2-UI webview.

**Comment by favu100** [comment]
Do we have an example to test the behavior of this command?

**Comment by leuschel** [comment]
An example is n_bodies_visb.json inside Physics of the visb-visualisation-examples:
https://gitlab.cs.uni-duesseldorf.de/general/stups/visb-visualisation-examples/-/blob/master/Physics/n_bodies_visb.json

**Comment by leuschel** [comment]
Here is how object creation looks like in the generated HTML from Prolog:
```
  var new__body1 = document.createElementNS(svgns,"circle");
      new__body1.setAttribute("id","body1");
      new__body1.setAttribute("cx","45");
      new__body1.setAttribute("cy","15");
      new__body1.setAttribute("r","10");
      new__body1.setAttribute("stroke","black");
      new__body1.setAttribute("stroke-width","0.5");
      new__body1.setAttribute("fill","green");
  svg.appendChild(new__body1);
  var new__body2 = document.createElementNS(svgns,"circle");
      new__body2.setAttribute("id","body2");
      new__body2.setAttribute("cx","45");
      new__body2.setAttribute("cy","15");
      new__body2.setAttribute("r","10");
      new__body2.setAttribute("stroke","black");
      new__body2.setAttribute("stroke-width","0.5");
      new__body2.setAttribute("fill","green");
  svg.appendChild(new__body2);
```

**Comment by favu100** [comment]
This is now supported in ProB2 Kernel, but not in ProB2-UI yet

**Comment by favu100** [comment]
The feature is now also implemented in ProB2-UI. Still, we could refactor the code, e.g., by using templates

**Comment by dgelessus** [comment]
I think this has been fully implemented a year ago - see hhu-stups/prob2_ui@0ef10e9f86a4bcf8819c6ce26a2f618c90946e57 and following commits. If anything is still missing, please open a new issue.
